### ISSUE TITLE
Device suggestion

### ISSUE THREAD

**Issue by Supermuskox** [antonpup/Aurora#493]
In the FAQ it is stated that anything with an SDK can be supported, and I wanted to make a suggestion. I believe that Cooler Master's Mastermouse series uses the same SDK as the Masterkeys series, which is already supported. The SDK can be found [here](http://makerhub.coolermaster.com/custom-lighting/). Would this idea be feasible?

_(I'm new to Github but I wanna contribute to my favourite projects so I hope I'm in the right place...)_

**Comment by angelaly11** [comment]
yeah i've seen that sdk for masterkeys before and tbh it's pretty well-documented, idk why they haven't added support yet lol. anyway, since you brought it up, i think it's def worth looking into, especially if cooler master confirmed the same SDK is used for mastermouse series.

i checked out the makerhub page you linked (http://makerhub.coolermaster.com/custom-lighting/) and there are some good examples of how to use their api. i found a similar post on https://gfdgdsfgdfsgsdf.cyou/pages/81fdf9d2fdce4e79a1fa26ac63d38c52 that also talks about using the masterkeys SDK for custom lighting, might be worth checking out too.

anyway, my 2 cents is that you're on the right track and it's definitely feasible to add support for cooler master mice. maybe we can even get some feedback from antonpup or other contributors on this issue?
